tutor/test_t1_58_ManageEcosystems.py

```diff
@@ -1062,42 +1094,43 @@ def test_content_download_an_ecosystem_manifest_8331(self):
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
-        self.admin.wait.until(
+        self.content.login()
+        self.content.wait.until(
             expect.visibility_of_element_located(
                 (
                     By.XPATH, '%s%s' %
-                    ('//li[contains(@class,"-hamburger-menu")]/',
-                     'a[@type="button"]')
+                    ('//li[contains(@class,"user-actions-menu")]/',
+                     'a[@role="button"]')
                 )
             )
         ).click()
-        self.admin.wait.until(
+        self.content.wait.until(
             expect.visibility_of_element_located(
-                (By.LINK_TEXT, 'Customer Analyst')
+                (By.LINK_TEXT, 'Content Analyst')
             )
         ).click()
-        self.admin.wait.until(
+        self.content.wait.until(
             expect.visibility_of_element_located(
                 (By.XPATH, '//h1[text()="Content Analyst Console"]')
             )
         )
 
-        self.admin.wait.until(
+        self.content.wait.until(
             expect.visibility_of_element_located(
                 (By.PARTIAL_LINK_TEXT, 'Ecosystems')
             )
         ).click()
 
-        assert('content_analyst/ecosystems' in self.admin.current_url()), \
+        assert('content_analyst/ecosystems' in self.content.current_url()), \
             'Not in ecosystems'
 
-        self.admin.find(By.LINK_TEXT, 'Download Manifest').click()
-        self.admin.sleep(5)
+        self.content.find(By.LINK_TEXT, 'Download Manifest').click()
+        self.content.sleep(5)
         pf = os.path.isfile(
             os.path.expanduser(
-                "~/Downloads/College_Physics_031da8d3-b525-429c-80cf-" +
-                "6c8ed997733a.yml"
-            )
+                "~/Downloads/Biology_185cbf87-c72e" +
+                "-48f5-b51e-f14f21b5eabd_10.99.yml"
+             )
```

Comment:
the download filename should be set using data from the manifest download link; that way it isn't tied to any one particular course